packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -240,7 +240,7 @@ const TileIconButton = React.forwardRef((props, forwardedRef) => {
   const { className, children, ...rest } = props;
   return (
     <IconButton
-      className={className}
+      className={cx('iui-tile-thumbnail-button', className)}
```

Comment:
I would suggest supporting both the new and old way, using a selector like this:

```css
.iui-tile-thumbnail-button,
:is(.iui-tile-thumbnail-type-indicator, .iui-tile-thumbnail-quick-action) button {
  …
}
```

Then we can "deprecate" passing a raw `IconButton` into `Tile.TypeIndicator`/`Tile.QuickAction` and automatically change it to `Tile.IconButton` in the migration tool.

Reply:
That output is wrong. The `button` should be a child of `iui-tile-thumbnail-type-indicator`. And `iui-tile-thumbnail-button` does not need to be nested.

Reply:
1. In the CSS demo we had:
```html
<button class="iui-button-base iui-button iui-field iui-tile-thumbnail-type-indicator">
  … 
</button>
```

2. In the React demo we had:
```html
<div class="iui-tile-thumbnail-type-indicator">
  <button class="iui-button-base iui-button iui-field">
      … 
  </button>
</div>
```

3. The new method is:
```html
<div class="iui-tile-thumbnail-type-indicator">
  <button class="iui-button-base iui-button iui-field iui-tile-thumbnail-button">
      … 
  </button>
</div>
```

Which of the 3 do we want to support?  I was thinking 1 & 3 but your response has me thinking 2 & 3.

Reply:
Supporting 1 & 3, CSS reduction as well in https://github.com/iTwin/iTwinUI/pull/2101/commits/8cb2894c84a804138b33985e374d57c29413ff68

Reply:
Yes, we want 2 & 3.

The reason I suggested a plain `button` selector is to make it _not_ dependent on the selectors in `iui-button`. Since this is now a deprecated API, I'm ok with there being some unintended side-effects in unsupported button types, as long as the borderless button looks ok. I doubt anyone would want to put anything other than a borderless button there anyway.

Reply:
Fixed in [`7020896`](https://github.com/iTwin/iTwinUI/pull/2101/commits/7020896d83a318f949baf26871300a9d582bb6db)